registration-processor/post-processor/registration-processor-printing-stage/src/main/java/io/mosip/registration/processor/print/stage/PrintingStage.java

```diff
@@ -383,6 +391,7 @@ private CredentialRequestDto getCredentialRequestDto(String regId,String issuerI
 
 		credentialRequestDto.setEncryptionKey(generatePin());
 		additionalAttributes.put("templateTypeCode",templateTypeCode);
+		additionalAttributes.put("registrationId", regId);
```

Comment:
this part, lets check with Rounak once to see if we are ok to add the registration id for all the partners

Reply:
As per discussion with Rounak, we might want to make this configurable, but that can handled as separate task

Reply:
ok.

Reply:
Create an separate task for dev team to handle this, so marking this comment as resolved
https://mosip.atlassian.net/browse/MOSIP-28128